data/ShopifyDemoData.xml

```diff
@@ -20,4 +20,5 @@ along with this software (see the LICENSE.md file). If not, see
     <moqui.service.message.SystemMessageRemote systemMessageRemoteId="DEMO_SHOPIFY_GET_COLLECTS_REMOTE" systemMessageTypeId="ShopifyGetCollectsMessage" sendUrl="https://70085f12deca125c586aaf311c198735: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@tailorsoft-test.myshopify.com/admin/api/2021-01/collects.json" prodcutStoreId="RCHERBALS" username="70085f12deca125c586aaf311c198735" password="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"/>
     <moqui.service.message.SystemMessageRemote systemMessageRemoteId="DEMO_SHOPIFY_DELETE_COLLECT_REMOTE" systemMessageTypeId="ShopifyDeleteCollectMessage" sendUrl="https://70085f12deca125c586aaf311c198735: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@tailorsoft-test.myshopify.com/admin/api/2021-01/collects/" prodcutStoreId="RCHERBALS" username="70085f12deca125c586aaf311c198735" password="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"/>
     <moqui.service.message.SystemMessageRemote systemMessageRemoteId="DEMO_SHOPIFY_GET_ORDERS_REMOTE" systemMessageTypeId="ShopifyGetOrdersMessage" sendUrl="https://70085f12deca125c586aaf311c198735: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@tailorsoft-test.myshopify.com/admin/api/2021-01/orders.json?status=" prodcutStoreId="RCHERBALS" username="70085f12deca125c586aaf311c198735" password="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"/>
+    <moqui.service.message.SystemMessageRemote systemMessageRemoteId="DEMO_SHOPIFY_SET_INVENTORY_LEVEL_REMOTE" systemMessageTypeId="ShopifySetInventoryMessage" sendUrl="https://70085f12deca125c586aaf311c198735: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@tailorsoft-test.myshopify.com/admin/api/2021-01/inventory_levels/set.json"                  prodcutStoreId="RCHERBALS" username="70085f12deca125c586aaf311c198735" password="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"/>
```

Comment:
Change DEMO_SHOPIFY_SET_INVENTORY_LEVEL_REMOTE for DEMO_SHOPIFY_SET_INVENTORY_REMOTE